frontend/src/app/modules/home/projects/projects-routing.module.ts

```diff
@@ -0,0 +1,25 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { CreateProjectComponent } from './create-project/create-project.component';
+import { EditComponent } from './edit/edit.component';
+import { ProjectsComponent } from './projects/projects.component';
+
+const routes: Routes = [{
+    path: '',
+    component: ProjectsComponent
+}, {
+    path: 'create',
+    component: CreateProjectComponent
+}, {
+    path: ':id',
+    component: EditComponent
+}, {
+    path: '',
+    redirectTo: 'projects'
```

Comment:
Redirect to path `projects` that doesn't exist in this routing module. Why is there 2 path with `' '`? 
Are you sure that path `:id` must redirect user to edit page? I think in the future it will be a some project-page. Maybe change this route to `edit/:id`.

Reply:
But at all, ш like that you have moved all projects to the module :)

Reply:
Fix it) and add new route for edit home/projects/edit/123